web-common/src/features/dashboards/pivot/pivot-utils.ts

```diff
@@ -516,25 +515,61 @@ export function getSortFilteredMeasureBody(
   return { sortFilteredMeasureBody, isMeasureSortAccessor, sortAccessor };
 }
 
+function getValuesForFlatTable(
+  tableData: PivotDataRow[],
+  rowDimensions: string[],
```

Comment:
Why are these the `rowDimensions`, not the `colDimensions`? Given the pills are labeled as "Columns":

<img width="607" alt="image" src="https://github.com/user-attachments/assets/5c652cfe-a490-4153-b090-37005e024764" />

Reply:
This is because the way implementation is set up for Pivot tables. Column dimension internally are treated as values on which measures would be split and hence have a very different way of working. For flat table we need just a basic query so it made sense to use rowDimensions as a proxy for implementing it. I have added a comment for this in `pivot-data-config`.